clients/da-vinci-client/src/main/java/com/linkedin/davinci/helix/LeaderFollowerPartitionStateModel.java

```diff
@@ -122,11 +121,11 @@ public void onBecomeStandbyFromOffline(Message message, NotificationContext cont
         }
         throw e;
       }
+      heartbeatMonitoringService
```

Comment:
I thought we moved this down to after waitConsumptionCompleted due to another issue?  Or am I misremembering?